lib/matplotlib/tests/test_backend_qt.py

```diff
@@ -157,6 +216,7 @@ def test_dpi_ratio_change():
     Make sure that if _dpi_ratio changes, the figure dpi changes but the
     widget remains the same physical size.
     """
+    return
```

Comment:
This doesn't look necessary.

Reply:
Ah yes, this test fails on my notebook as the Qt does not have enough time to open the window. Its not my fault, I guess, and I can suggest an easy fix for that. It will be included in the next commit.